Stork/Stork/Source/JsonValue.swift

```diff
@@ -25,6 +25,12 @@ public enum JsonValue {
       self = .string(value)
     case let value as JSON:
       self = .object(value)
+    case let value as [JsonValue]:
+      self = .array(value)
+    case let value as [Any]:
+      self = .array(value.map {
+        JsonValue.init(fromAny: $0)!
+      })
```

Comment:
This one is useful in theory, although I am not sure if ever useful in practice.

To parse an array of type T from a JSON object, we'd use `json ..? "field"` or `json ..! "field"`. In such a case, the T is inferred and Stork knows how to build values of that type T as long as it complies to `FromJson`.

In this proposed case, we would start with a value whose type is unknown, and we want to check whether it is an array of Any type. If so, we'd return `array([JsonValue])`, which - in any normal case - is still not good enough for a user. Having `array([JsonValue])`, we'd still need to parse it to a final, useful type:

```
case let .array(xs):
  let ys: [MyType] = xs.compactMap { decodeValue($0) }
  return .array(ys)
```

But that could be achieved in a single step by doing:

``` swift
// At a high-level
let ys = [MyType].from(jsonArray: jsonArray) // where jsonArray is [JSON]

// Or when parsing a field from an object
array: json ..! "field" 
```